src/v2/ion/ViewClassNamesFactory.js

```diff
@@ -0,0 +1,88 @@
+/* eslint complexity: [2, 25] */
+import {FORMS} from './RemediationConstants';
+
+const FORMNAME_CLASSNAME_MAPPINGS = {
+  [FORMS.IDENTIFY]: {
+    'identify': ['identify', 'primary-auth'],
+    'password': ['identify', 'primary-auth']
+  },
+  [FORMS.ENROLL_PROFILE]: {
+    'enroll-profile': ['enroll-profile', 'registration'],
```

Comment:
use `_` as default ?

Reply:
I tried that approach, it required me to define a formName entry for every form. This is an edge case for when you have more than one class name and dont have a factor, only for those cases we use the key as formname

Reply:
I see it's kind of similar. basically need to have `let type = '_'` in the following method.

Reply:
- FORMNAME_CLASSNAME_MAPPINGS does not currently have an entry for every formName example success, terminal etc. 
- For those cases that are not here we are assuming that there is no type so we just return [formName] as the class.
- If we use type as `_`,  we would need to add an entry here for that formName with the type as `_` ?